style.css

```diff
@@ -1,336 +1,704 @@
+  }
 
-.ab-li {
-  padding: 0;
-}
+  .final {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
 
-.ab-title {
-  margin-top: 10px;
-  padding-top: 0;
-  color: #344563;
-}
+  .ab-lang {
+    background: #ebf0ee;
+    height: 352px;
+    margin-top: 50px;
+    margin-left: 25px;
+  }
 
-.ab-head {
-  margin-top: 60px;
-  padding-top: 0;
-  color: #344563;
-  font-size: 40px;
-  margin-left: 100px;
-}
+  .list {
+    list-style: none;
+  }
 
-.ab-img {
-  margin-top: 20px;
-}
+  a {
+    color: #172b4d;
+  }
 
-.close {
-  margin-left: 25px;
-  margin-top: 50px;
-  background: #fff;
-  border: none;
-}
+  .lang {
+    width: 320px;
+  }
 
-.closure {
-  font-weight: 700;
-  font-size: 40px;
-  line-height: 52px;
-  text-align: center;
-  margin-right: 25px;
-}
+  /* ************************ section Three Above **************** */
 
-.ab-form {
-  margin-left: 25px;
-}
+  .prologue {
+    padding-left: 100px;
+    height: 1434.3px;
+  }
 
-#abt-contact {
-  display: flex;
-  flex-direction: column;
-}
+  .p-lang {
+    padding: 0;
+    display: flex;
+    justify-content: center;
+  }
 
-input[type="text"] {
-  height: 3rem;
-  width: 20rem;
-  margin-bottom: 2rem;
-  border-radius: 6px 6px 6px 6px;
-  border: #cfd0d1 solid 1px;
-  padding-left: 1rem;
-  font-size: 1rem;
-}
+  .btn-btn {
+    height: 35px;
+    font-size: 15px;
+    border-radius: 4px 4px 4px 4px;
+    margin: 3px;
+    margin-top: 10px;
+    color: #3a4a42;
+    background: #ebf0ee;
+    border: none;
+  }
 
-input[type="email"] {
-  height: 3rem;
-  width: 20rem;
-  margin-bottom: 2rem;
-  border-radius: 6px 6px 6px 6px;
-  border: #cfd0d1 solid 1px;
-  padding-left: 1rem;
-  font-size: 1rem;
-}
+  .btn {
+    margin-top: 15px;
+    height: 50px;
+    width: 128px;
+    background: #36b37f;
+    border-radius: 6px 6px 6px 6px;
+    border: none;
+    display: flex;
+    color: #fff;
+    font-size: 19px;
+    margin-left: 100px;
+    margin-right: 100px;
+    padding-top: 14px;
+    padding-left: 13px;
+  }
 
-input[type="submit"] {
-  padding-bottom: 1rem;
-  margin-top: 30px;
-}
+  .mob-img {
+    display: none;
+  }
 
-.footer {
-  display: flex;
-  height: 10px;
-}
+  .ab-btn-btn {
+    margin-right: 90px;
+    width: 160px;
+    margin-left: 0;
+    padding-left: 10px;
+    margin-bottom: 12px;
+  }
 
-.footer-img {
-  margin-left: 70px;
-  margin-bottom: 150px;
-}
+  .ab-btn {
+    background: #fff;
+    color: #36b37e;
+    display: flex;
+    padding-left: 15px;
+    padding-right: 10px;
+    padding-top: 10px;
+  }
 
-.ab-social {
-  margin-top: 50px;
-}
+  .flex-tool {
+    display: flex;
+    flex-direction: row;
+    flex-wrap: wrap;
+    align-items: center;
+    justify-content: center;
+  }
 
-.ab-soc {
-  font-size: 25px;
-  margin-right: 40px;
-  color: #42526e;
-}
+  .contact-me {
+    margin-top: 100px;
+    margin-right: 25px;
+    margin-left: 25px;
+    display: flex;
+    justify-content: space-between;
+  }
```

Comment:
Kindly adjust the contact me section to match the design. From the design, the form takes up a little above half of the available space while the text content take up the other.

You seem to have given a larger size of margin than the design shows.

Your design:
![image](https://user-images.githubusercontent.com/20554583/200075937-cbbc5768-548b-4202-a5ca-6e6d13cd2bfd.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/200075957-b7fbbdb1-650c-4ec7-9036-35e084e5a517.png)